client/state/stored-cards/test/reducer.js

```diff
@@ -20,8 +20,13 @@ import {
 } from 'state/action-types';
 import reducer, { items } from '../reducer';
 import { STORED_CARDS_FROM_API } from './fixture';
+import { useSandbox } from 'test/helpers/use-sinon';
 
 describe( 'items', () => {
+	useSandbox( ( sandbox ) => {
+		sandbox.stub( console, 'warn' );
```

Comment:
Yup. This gets pretty noisy in the test suite. We can still spy to see if the stub has been called, but we already have pretty good coverage.